modules/ROOT/pages/microsoft-sharepoint-2013-connector.adoc

```diff
@@ -58,7 +70,7 @@ Supported Microsoft SharePoint Online authentication:
 Configuring the connector to use claims-based authentication requires the following information from the Security Token Service (STS) that to use for authenticating:
 
 * *Sts Url*: The STS endpoint that accepts username and password for authenticating users and understands the WS-Trust 1.3 protocol. When the STS is Microsoft’s ADFS (Active Directory Federation Services), this URL usually is `https://youradfs.com/adfs/services/trust/13/usernamemixed`.
-* *Sts App Identifier*: A string that identifies the Sharepoint site in the STS. It is also known as Relying Party Identifier, Client Identifier, Scope, or Realm. When the STS is Microsoft’s ADFS, this value can be discovered in the AD FS Management console under AD FS at *Trust Relationships* > *Relying Party Trusts* > _(Sharepoint Site’s relying part properties)_ > *Identifiers* tab.
+* *Sts App Identifier*: A string that identifies the Sharepoint site in the STS. It is also known as Relying Party Identifier, Client Identifier, Scope, or Realm. When the STS is Microsoft’s ADFS, this value can be discovered in the AD FS Management console under AD FS at *Trust Relationships* > *Relying Party Trusts* > (Sharepoint Site’s relying part properties) > *Identifiers* tab.
```

Comment:
```suggestion
* *Sts App Identifier*: A string that identifies the Sharepoint site in the STS
It is also known as Relying Party Identifier, Client Identifier, Scope, or Realm. When the STS is Microsoft’s ADFS, this value can be discovered in the AD FS Management console under AD FS at *Trust Relationships* > *Relying Party Trusts* > (Sharepoint Site’s relying part properties) > *Identifiers* tab.
```
Do you think that the audience will understand the click-flow here, with the text in parens?